CHANGELOG.md

```diff
@@ -35,13 +60,7 @@ All notable changes to this project will be documented in this file.
 
 ### :boom: Breaking changes
 - Update node engines to next LTS (node 20 / npm 9)
-- The deprecated `NcMultiselect` and `NcMultiselectTags` components were removed [\#4281](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4281) ([susnux](https://github.com/susnux))
-  - Migration: Use the `NcSelect` and `NcSelectTags` components instead
-- The deprecated `NcPopoverMenu` component was removed [\#4081](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4081) ([raimund-schluessler](https://github.com/raimund-schluessler))
-  - Migration: Use the `NcActions` component instead
-- The deprecated `NcAppNavigationCounter` component was removed [\#4096](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4096) ([raimund-schluessler](https://github.com/raimund-schluessler))
-  - Migration: Use the `NcCounterBubble` component instead
-- The deprecated `excludeClickOutsideClasses` property was removed from `clickOutsideOptions` [\#4088](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4088) ([raimund-schluessler](https://github.com/raimund-schluessler))
```

Comment:
Shouldn't we keep these?

Reply:
I have no clue what is happening there... reset and recreate was the only thing fixing it...